doc/contributing/contribute_code.md

```diff
@@ -10,19 +10,15 @@ project. For how to contribute to the documentation, see
 
 Make sure you have a Java SDK available to compile the application.
 
-The project has moved to the [Eclipse Temurin Java SDK](https://adoptium.net) 
+The project has moved to the [Apache Adoptium Java SDK](https://adoptium.net) 
 (previously AdoptOpenJDK) for distribution builds.
 
-z390 is using java 21 to build the package but code must support Java 1.8.
-
-To ensure alignment with other developers, use of a Java SDK 21 or greater
-is recommended. At a minimum, you will require an SDK of 9 or greater 
-as the build jobs use parameters not available in earlier versions.
+z390 requires Java 1.8 or greater.
```

Comment:
Why are you replacing lines 16-20 with this one line?

Reply:
change reverted